stock_return_request/models/stock_return_request.py

```diff
@@ -267,7 +264,7 @@ def _action_confirm(self):
                     return_move.product_uom_qty += qty
                 done_moves.setdefault(move, self.env["stock.move"])
                 done_moves[move] += return_move
-                return_move._action_confirm()
+                return_move.state = "confirmed"
```

Comment:
Why _action_confirm() should not be used anymore ? At least a comment should be set